### PR TITLE
Update test workflow and run on all pushes

### PR DESCRIPTION
**Pull request by colinking**
Noticed that tests were not running on stacked PRs. While fixing that, I updated our test workflow to match airport.

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #421** <a href="https://app.graphite.dev/github/pr/airplanedev/cli/421" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/cli/421?utm_source=stack-comment).

**Comment by colinking** [comment]
@francescaleung: can you 👀 today? 🙏🏻

File: pkg/api/watcher.go [hunk 1]
```diff
@@ -126,25 +127,37 @@ func (w *Watcher) send(ctx context.Context, state RunState) {
...
 
 	eg.Go(func() error {
 		run, err := w.client.GetRun(subctx, w.runID)
 		if err != nil {
 			return errors.Wrap(err, "get run")
 		}
 
-		state.Status = run.Run.Status
+		mu.Lock()
+		stopped := state.Stopped()
```
Comment (francescaleung) [inline pkg/api/watcher.go:143]: should `run.Run.Status` get assigned to `state.Status` before this line?

**Review by francescaleung — APPROVED** [review]

File: pkg/print/table.go [hunk 2]
```diff
@@ -201,7 +203,7 @@ func (t Table) outputs(outputs api.Outputs) {
 
 // formatOutputName converts output_name -> Output Name.
 func formatOutputName(key string) string {
-	return strings.Title(strings.ReplaceAll(key, "_", " "))
+	return cases.Title(language.AmericanEnglish).String(strings.ReplaceAll(key, "_", " "))
```
[the remark this answers is not in the document]
Comment (colinking) [reply pkg/print/table.go:206]: Good catch! Will update.

CleanShot 2022-05-03 at 15 27 38

File: pkg/api/watcher.go (hunk 1 above)
Comment (colinking) [reply pkg/api/watcher.go:143]: It happens down on line 157. Same behavior!

**Pull request merged**